.github/workflows/update-cmo-jsonnet-deps.yaml

```diff
@@ -0,0 +1,58 @@
+name: Upgrade cmo jsonnet dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'
+env:
+  USER: 'github-actions[bot]<github-actions[bot]@users.noreply.github.com>'
+
+jobs:
+  jsonnet-update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: openshift/cluster-monitoring-operator
```

Comment:
Just a thought ... Is this going to be the same for all projects? if that is the case, may be the entire workflow can be defined as an action.

Reply:
It will be different per project.

For other components, (e.g. Thanos, Prometheus) upstream repo also would get involved along with `git merge` step.